### PR TITLE
[-] Command Generate - Migrate all options from environment variables to command parameters

### PR DESCRIPTION
**Pull request by rap2hpoutre**
Full PR including #260 (thanks to @VincentMolinie :wave:), #269, #271, #273, #276. @arnaudbesnier Final step, everything was already reviewed inside. Still it may require one last review :)

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
[Lumber - Replace all environment variables by options. And then adapt the onboarding help & commands.](https://trello.com/c/7ckuvEHx/3728-lumber-replace-all-environment-variables-by-options-and-then-adapt-the-onboarding-help-commands)

File: services/prompter.js [hunk 1]
```diff
@@ -112,6 +112,7 @@ async function Prompter(program, requests) {
     }
   }
 
+
```
Comment (arnaudbesnier) [inline services/prompter.js]: [Cleaning] What is this new line for? (Maybe the linter could prevent more than 1 newline in a raw)

Comment (rap2hpoutre) [reply services/prompter.js]: > Maybe the linter could prevent more than 1 newline in a raw

Yes: https://eslint.org/docs/rules/no-multiple-empty-lines I will create another PR for that.

> What is this new line for?

:) Useless (maybe a merge glitch). Fixed now.

**Comment by rap2hpoutre** [comment]
@arnaudbesnier OK, I answered your comment and fixed it. **Do you have a suggestion for the final squashed commit message?** Currently it's "Improvement/full options" (name of the branch).

**Comment by arnaudbesnier** [comment]
@rap2hpoutre my suggestion is the new PR title. Feel free to change it, if you're not ok with it.
You can merge this whenever you want.

**Pull request merged**